doc/source/user_guide/timeseries.rst

```diff
@@ -1383,7 +1383,7 @@ For some frequencies you can specify an anchoring suffix:
     "(B)Q(S)\-SEP", "quarterly frequency, year ends in September"
     "(B)Q(S)\-OCT", "quarterly frequency, year ends in October"
     "(B)Q(S)\-NOV", "quarterly frequency, year ends in November"
-    "(B)A(S)\-DEC", "annual frequency, anchored end of December. Same as 'A'"
+    "(B)A(S)\-DEC", "annual frequency, anchored end of December. Same as 'Y'"
```

Comment:
should these say "(B)Y(S)\-DEC" (and similarly for other months)?

Reply:
Thank you, that's correct it should say "(B)Y(S)-DEC", I missed that. Now it's all good.